scripting/shavit-zones.sp

```diff
@@ -1402,9 +1497,18 @@ public void SQL_InsertZone_Callback(Database db, DBResultSet results, const char
 
 		return;
 	}
+	
+	
+	if(gMZ_Type[data] == Zone_CustomSpawn)
```

Comment:
2 newlines shouldn't happen consecutively. I think it happens multiple times in this function